pyproject.toml

```diff
@@ -25,7 +25,7 @@ prow-jobs-scraper = "prowjobsscraper.main:main"
 
 [project.optional-dependencies]
 test-runner = ["tox==3.26.0"]
-unit-tests = ["pytest==7.1.2", "pytest-cov==3.0.0", "pytest-httpserver==1.0.4"]
+unit-tests = ["pytest==7.2.0", "pytest-cov==3.0.0", "pytest-httpserver==1.0.4"]
```

Comment:
@adriengentil maybe it will be good to have it in separate lines so that PRs that update the same section won't need to rebase and lose lgtm label